src/acl/external/kerberos_ldap_group/support_sasl.cc

```diff
@@ -241,7 +241,7 @@ tool_sasl_bind(LDAP * ld, char *binddn, char *ssl)
      * char  *sasl_secprops = (char *)"maxssf=56";
      * char  *sasl_secprops = NULL;
      */
-    struct berval passwd = {0, NULL};
+    struct berval passwd = { };
```

Comment:
IIRC, this PR does not use a space in most cases:

```suggestion
    struct berval passwd = {};
```